### PR TITLE
Add tests for Routes components

### LINKED ISSUE #5751: Add tests for the newly added `Routes` components
Two components `Routes` have been added in #5652. We can probably add smoke tests for each of them, like:

- ensuring each component has a certain number of `Route`
- some routes are `config`-dependent, so we could test that as well

### PR DESCRIPTION
**Pull request by willdurand**
Fix #5751

---

Added a bunch of new test cases.

### CODE REVIEW

File: src/amo/components/Routes/index.js [hunk 1]
```diff
@@ -25,9 +25,14 @@ import UserProfileEdit from 'amo/components/UserProfileEdit';
 import SimulateAsyncError from 'core/components/error-simulation/SimulateAsyncError';
 import SimulateClientError from 'core/components/error-simulation/SimulateClientError';
 import SimulateSyncError from 'core/components/error-simulation/SimulateSyncError';
+import type { ConfigType } from 'core/types/config';
+
+type Props = {|
```
Comment (rebmullin) [inline src/amo/components/Routes/index.js]: hmm not sure why this looks cut off to me...? It looks okay in my editor..

Comment (willdurand) [reply src/amo/components/Routes/index.js]: Probably an issue with the "Refined GitHub" extension due to some GitHub changes (seeing the same).

File: tests/unit/disco/components/TestRoutes.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import NotFound from 'core/components/ErrorPage/NotFound';
+import Routes from 'disco/components/Routes';
+import { getFakeConfig } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const render = (props = {}) => {
+    return shallow(<Routes {...props} />);
+  };
+
+  it('renders the routes for the amo app', () => {
```
Comment (rebmullin) [inline tests/unit/disco/components/TestRoutes.js]: Should this say "for disco"?

File: src/amo/components/Routes/index.js (hunk 1 above)
Comment (rebmullin) [reply src/amo/components/Routes/index.js]: ahh ok...

File: tests/unit/disco/components/TestRoutes.js (hunk 2 above)
Comment (willdurand) [reply tests/unit/disco/components/TestRoutes.js]: it should!

**Pull request merged**